scripts/fetch/dv_data.R

```diff
@@ -6,6 +6,7 @@ fetch.dv_data <- function(viz = getContentInfo(viz.id = "NM_data")){
   site.id <- viz[["siteID"]]
   
   dailyData <- readNWISdv(site.id, "00060","","")
+  dailyData <- renameNWISColumns(dailyData)
```

Comment:
I was trying to make the `fetch` as totally vanilla as possible. To me, this would be a part of processing.